src/settings.js

```diff
@@ -47,11 +47,14 @@ var INVOKE_RUN = 1; // Whether we will run the main() function. Disable if you e
 var NO_EXIT_RUNTIME = 0; // If set, the runtime is not quit when main() completes (allowing code to
                          // run afterwards, for example from the browser main event loop).
 var MEM_INIT_METHOD = 0; // How to represent the initial memory content.
-                         // 0: keep array literal representing the initial memory data
+                         // 0: embed a base64 string literal representing the initial memory data
                          // 1: create a *.mem file containing the binary data of the initial memory;
                          //    use the --memory-init-file command line switch to select this method
                          // 2: embed a string literal representing that initial memory data
-                         //    XXX this is known to have bugs on windows, see https://github.com/kripken/emscripten/pull/3326
+                         //    XXX this is known to have bugs on windows, see https://github.com/kripken/emscripten/pull/3326;
+                         //    due to mode 0's recent upgrade to space-efficient base64 encoding
+                         //    and the long-standing issues with this mode, this mode is now
+                         //    deprecated in favor of mode 0
```

Comment:
Based on what you said [here](https://github.com/kripken/emscripten/pull/5296#discussion_r122568346) (to correct my mistaken assumption that `MEM_INIT_METHOD=2` was also doing base64) and what I read up on afterwards, I believe `MEM_INIT_METHOD=2`'s encoding is unambiguously more space-efficient than base64 and the delta wouldn't be changed by a merge/rebase.

I made this change after @kripken suggested [here](https://github.com/kripken/emscripten/pull/5296#discussion_r122785031) that I add the deprecation note, which I'd assumed was related to whatever compatibility issues had prevented it from replacing `MEM_INIT_METHOD=0` up until now (in that base64 is now "good enough" while having no such issues).

However, he'd just misread your response to my bringing up the option of deprecation as an agreement. I'll leave this up to you guys to decide.